src/System.Drawing.Common/src/System/Drawing/Graphics.cs

```diff
@@ -4846,9 +4843,7 @@ public static IntPtr GetHalftonePalette()
             return s_halftonePalette;
         }
 
-        //This will get called for ProcessExit in case for WinNT..
-        //This will get called for ProcessExit AND DomainUnLoad for Win9X...
-        //
+        // This is called from AppDomain.ProcessExit and AppDomain.DomainUnload
```

Comment:
The old comment was wrong.
The case is false for Win9X, as `Environment.OSVersion.Platform == Platform.Id.Win32Windows` in Win9x
```
if (Environment.OSVersion.Platform != System.PlatformID.Win32Windows))
{
   AppDomain.CurrentDomain.DomainUnload += new EventHandler(OnDomainUnload);
}
```